src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -132,6 +135,28 @@ public void testWithMergingView() throws Exception {
         assertTrue(mvcResult.getResponse().getContentAsString().startsWith("<?xml version=\"1.0\" encoding=\"UTF-8\"?>"));
     }
 
+    @Test
+    public void testExportSv() throws Exception {
+        Path path = Paths.get(getClass().getClassLoader().getResource(MICRO_GRID_BE_ZIP).toURI());
```

Comment:
Instead of duplicating the CGMES file in this project you should use the "cgmes-conformity" module provided by powsybl core and that already contains this network
```
        Network network = new CgmesImport()
                .importData(CgmesConformity1ModifiedCatalog.microGridBaseCaseBE().dataSource(), null);
```

Reply:
done